evap/staff/importers.py

```diff
@@ -574,55 +574,45 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
                 evaluation.participants.add(*participants)
             self.create_success_messages(students_created, responsibles_created)
 
+    @staticmethod
+    def make_phrase(count, singular, plural):
+        count_word = pgettext("count", "no") if count == 0 else str(count)
+        return f"{count_word} {ngettext(singular, plural, count)}"
+
     def create_success_messages(self, students_created, responsibles_created):
-        student_count = len(students_created)
-        student_word = pgettext("count", "no") if student_count == 0 else str(student_count)
-        responsible_count = len(responsibles_created)
-        responsible_word = pgettext("count", "no") if responsible_count == 0 else str(responsible_count)
-        evaluation_count = len(self.evaluations)
-        evaluation_word = pgettext("count", "no") if evaluation_count == 0 else str(evaluation_count)
-
-        student_phrase = ngettext("{count} student", "{count} students", student_count).format(count=student_word)
-        responsible_phrase = ngettext("{count} responsible", "{count} responsibles", responsible_count).format(
-            count=responsible_word
-        )
-        evaluation_phrase = ngettext(
-            "{count} course/evaluation", "{count} courses/evaluations", evaluation_count
-        ).format(count=evaluation_word)
+        if not students_created and not responsibles_created and not self.evaluations:
```

Comment:
codecov says this case is never true in tests, a test for this should be easy

Reply:
(Codecov says the `if` is not covered, but the if will always be covered, only the return might not be covered. So, maybe the if and the return are actually both covered. See #1723 for details.)

Reply:
Depending on how the importers work, this is never really reached. If the evaluations always get made

Reply:
So if we upload an empty file, it will result in an error earlier?

@janno42 On merge situations, this is basically the same as https://github.com/e-valuation/EvaP/pull/1683#issuecomment-1115302197 -- `self.evaluations` might contain evaluations where nothing will be changed, due to the merge.